src/playhead-layer.js

```diff
@@ -282,5 +281,16 @@ define([
     }
   };
 
+  PlayheadLayer.prototype.setTimeLabelPrecision = function() {
```

Comment:
As we're not passing a new precision value to this function, i'd suggest renaming this as `updatePlayheadText()`